be/src/common/config.h

```diff
@@ -758,6 +757,8 @@ CONF_Int32(connector_io_tasks_adjust_interval_ms, "50");
 CONF_Int32(connector_io_tasks_adjust_step, "1");
 CONF_Int32(connector_io_tasks_adjust_smooth, "4");
 CONF_Int32(connector_io_tasks_slow_io_latency_ms, "50");
+CONF_mDouble(scan_use_query_mem_ratio, "0.25");
+CONF_Double(connector_scan_use_query_mem_ratio, "0.3");
```

Comment:
it's better to add these configurations into session variable so that we can change them by avoiding  to restart BE

Reply:
@silverbullet233 done. I've added another session variable for that.